scenes/splash/splash.gd

```diff
@@ -0,0 +1,15 @@
+extends Node2D
+
+
+func _process(_delta: float) -> void:
+	if Input.is_action_just_pressed("ui_accept"):
+		switch_to_intro()
+
+
+func switch_to_intro() -> void:
+	get_tree().change_scene_to_packed(preload("res://scenes/intro/intro.tscn"))
+
+
+func _on_logo_stitcher_finished() -> void:
+	await get_tree().create_timer(5.0).timeout
+	switch_to_intro()
```

Comment:
I've seeing an error on the `Output` tab because the `LogoStitcher` (which runs in the editor because it's a `@tool`) tries to call `_on_logo_stitcher_finished` through the signal `finished`. We could avoid this error by adding:

```gdscript
func _ready() -> void:
   $LogoStitcher.finished.connect(self._on_logo_stitcher_finished)
```

so the connection to the signal doesn't exist in the editor.
Another option is adding:
```gdscript
if Engine.is_editor_hint(): return
```
in the body of `_on_logo_stitcher_finished`, but I think we can just keep this node as it is and get the signal connected through code.

![image](https://github.com/user-attachments/assets/ef317c05-51f5-4af8-b945-7af5145e330e)

Reply:
I decided the tool script is still useful so I moved the signal connection to code as you suggest.